### ISSUE TITLE
IoP Wallet - Aviso "Ayuda" en pantalla de enviar 

### ISSUE THREAD

**Issue by maydiamc** [Fermat-ORG/beta-testing-program#853]
#### Nombre pantalla
Enviar (send coins)

### Nombre de la Aplicación 
IoP Wallet Android

-----

##### Reporte de error: 

   *Se recomienda colocar cerrar en vez de rechazar, en el botón de la parte inferior. 

##### Screenshot o Capture de pantalla:

![screenshot_1](https://cloud.githubusercontent.com/assets/17274154/19156111/bfb14c0c-8bae-11e6-956a-0428be2d8b5a.png)

##### Logcat

[iop Ayudas.zip](https://github.com/Fermat-ORG/beta-testing-program/files/513981/iop.Ayudas.zip)

##### Pasos exactos para reproducir el error:

- Ingresar a la pantalla de enviar
- Dar click en los tres puntos
- Dar click en el botón ayuda

------

#### Dispositivo desde donde se realizo la prueba

- [x] Emulador
- [] Teléfono Físico
- Modelo de dispositivo: nexus 4
- Versión de Android: 5.1
- API: 22
- dpi: 240

##### Screenshot o Capture de pantalla:

![nexus 4 5 1](https://cloud.githubusercontent.com/assets/17274154/19156168/fc07a232-8bae-11e6-8598-b2a8b0a86779.png)

**Comment by maydiamc** [comment]
@furszy

**Comment by AdryCanonico** [comment]
@furszy  El caso es una experiencia de usuario. Cuando estén resueltos nosotras lo cerramos. Disculpa el inconveniente.

**Comment by furszy** [comment]
No, este caso no es experiencia del usuario ya que ustedes no saben que está pantalla posee activo el NFC..
